README.md

```diff
@@ -250,16 +250,16 @@ $ INGRESS_HOST=`minikube ip`
 $ echo $INGRESS_HOST
 ```
 
-Fill config files with the INGRESS_HOST in k8s/overlays/local/ :
+Fill config files with the INGRESS_HOST in k8s/live/local/ :
 ```bash
-$ sed -i -e "s/<INGRESS_HOST>/${INGRESS_HOST}/g" k8s/overlays/local/*
+$ sed -i -e "s/<INGRESS_HOST>/${INGRESS_HOST}/g" k8s/live/local/*
```

Comment:
dosent work cause there's a sub dir patches, maby we change it by using this cmd : 
`$ find k8s/live/local -type f -exec sed -i -e "s/<INGRESS_HOST>/${INGRESS_HOST}/g" {} \;`